### PR TITLE
Release pipelines including incubator repo

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

File: jenkins-jobs/scripts/deploy-snapshots.groovy [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+if (
+    !DEBEZIUM_REPOSITORY ||
+    !DEBEZIUM_BRANCH ||
+    !DEBEZIUM_INCUBATOR_REPOSITORY ||
+    !DEBEZIUM_INCUBATOR_BRANCH
+) {
+    error 'Input parameters not provided'
+}
+
+GIT_CREDENTIALS_ID = '17e7a907-8401-4b7e-a91b-a7823047b3e5'
```
Comment (gunnarmorling) [inline jenkins-jobs/scripts/deploy-snapshots.groovy]: Is this something secret?

File: jenkins-jobs/scripts/deploy-snapshots.groovy [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    stage ('Build and deploy Debezium') {
+        dir(DEBEZIUM_DIR) {
+            sh "mvn clean deploy -U -s $HOME/.m2/settings-snapshots.xml -DdeployAtEnd=true -DskipITs -DskipTests -Passembly"
+        }
+    }
+
+    stage ('Build and deploy Debezium Incubator') {
+        dir(ORACLE_ARTIFACT_DIR) {
+            sh "mvn install:install-file -DgroupId=com.oracle.instantclient -DartifactId=ojdbc8 -Dversion=12.1.0.2 -Dpackaging=jar -Dfile=ojdbc8.jar"
```
Comment (gunnarmorling) [inline jenkins-jobs/scripts/deploy-snapshots.groovy]: Maybe an env VAR / constant for the version?

File: jenkins-jobs/scripts/release-pipeline.groovy [hunk 3]
```diff
@@ -167,24 +216,32 @@ node('Slave') {
         def version = RELEASE_VERSION.split('\\.')
         IMAGE_TAG = "${version[0]}.${version[1]}"
         echo "Images tagged with $IMAGE_TAG will be used"
+        dir(ORACLE_ARTIFACT_DIR) {
+            sh "mvn install:install-file -DgroupId=com.oracle.instantclient -DartifactId=ojdbc8 -Dversion=12.1.0.2 -Dpackaging=jar -Dfile=ojdbc8.jar"
```
Comment (gunnarmorling) [inline jenkins-jobs/scripts/release-pipeline.groovy]: Same here. It'd be nice to somehow centralize this version number.

File: jenkins-jobs/scripts/deploy-snapshots.groovy (hunk 1 above)
Comment (jpechane) [reply jenkins-jobs/scripts/deploy-snapshots.groovy]: @gunnarmorling It is a handle to the Jenkins credentials and secrets store

File: jenkins-jobs/scripts/deploy-snapshots.groovy (hunk 2 above)
Comment (jpechane) [reply jenkins-jobs/scripts/deploy-snapshots.groovy]: @gunnarmorling Fixed

**Comment by gunnarmorling** [comment]
Thx!

**Pull request merged**